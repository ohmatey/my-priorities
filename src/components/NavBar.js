import React from 'react';

const NavBar = () => (
  <nav className="nav">
    <div className="nav-left">
      <a className="nav-item">
        My Priorities
      </a>
    </div>
  </nav>
);

export default NavBar;
