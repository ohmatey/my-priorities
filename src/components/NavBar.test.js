import React from 'react'
import { shallowToJson } from 'enzyme-to-json'

import NavBar from './NavBar'

describe('NavBar component', () => {
  it('matches snapshot', () => {
    const tree = shallowToJson(
      <NavBar />
    )
    expect(tree).toMatchSnapshot()
  })
})
