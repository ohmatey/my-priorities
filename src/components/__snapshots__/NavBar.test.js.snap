// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar component matches snapshot 1`] = `undefined`;
